src/main/java/com/ucm/ms/accounts/controllers/CreditCardController.java

```diff
@@ -1,40 +1,99 @@
 package com.ucm.ms.accounts.controllers;
 
-import com.ucm.ms.accounts.entities.CreditCard;
-import com.ucm.ms.accounts.services.CreditCardSearch;
+import java.util.Collection;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.Collection;
+import com.ucm.lib.dao.UserRepository;
+import com.ucm.ms.accounts.dao.AccountTypeDAO;
+import com.ucm.ms.accounts.dao.CreditCardDAO;
+import com.ucm.ms.accounts.dao.UserAccountDAO;
+import com.ucm.ms.accounts.entities.AccountType;
+import com.ucm.ms.accounts.entities.CreditCard;
+import com.ucm.ms.accounts.services.CreateCardService;
+import com.ucm.ms.accounts.services.CreditCardSearch;
 
 @CrossOrigin
 @RestController
 @RequestMapping("/creditcards")
 public class CreditCardController {
 	private final CreditCardSearch creditCardSearch;
+	
+	
+	@Autowired
+	AccountTypeDAO accountTypeDao;
+	
+	@Autowired
+	UserRepository userRepository;
+	
+	@Autowired
+	UserAccountDAO userAccountDao;
+	
+	@Autowired
+	CreditCardDAO creditCardDao;
+	
+	@Autowired
+	CreateCardService createCardService;
+	
 
 	@Autowired
 	public CreditCardController(CreditCardSearch creditCardSearch) {
 		this.creditCardSearch = creditCardSearch;
 	}
 
-	/**
-	 * GET /api/creditcards - Return all credit cards
-	 * 
-	 * @return All credit cards
-	 */
-	@GetMapping
-	public ResponseEntity<Collection<CreditCard>> get() {
-		try {
-			Collection<CreditCard> cards = creditCardSearch.getAll();
-			return new ResponseEntity<>(cards, HttpStatus.valueOf(200));
-		} catch (Exception e) {
-			return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+	
+		/**
+		 * GET /api/creditcards - Return all credit cards
+		 * 
+		 * @return All credit cards
+		 */
+	
+	
+		@GetMapping(path="get_credit_cards")
```

Comment:
I know I commented on this method before and is correct, but this path isn't really necessary. The previous implementation of this method is a GET under "/api/creditcards". This one is "/api/creditcards/get_credit_cards". There are a few other instances of this same thing happening in the PR, but yeah I don't think we should change it, as this was how all of the other entity URIs were defined. Also it just looks better tbh.